File: Examenes/EX2/22-1/BackEnd_CSharp/ConferenceSoftNETRemoting/ConferenceSoftModel/Evento.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ConferenceSoftModel
{
    public class Evento
    {
        private int _idEvento;
        private BindingList<Ponencia> _ponencias;
        private string _nombre;
        private DateTime _fecha;
        private string _descripcion;
        private byte[] _portada;
        private bool _activo;

        public int IdEvento { get => _idEvento; set => _idEvento = value; }
        public BindingList<Ponencia> Ponencias { get => _ponencias; set => _ponencias = value; }
        public string Nombre { get => _nombre; set => _nombre = value; }
        public DateTime Fecha { get => _fecha; set => _fecha = value; }
        public string Descripcion { get => _descripcion; set => _descripcion = value; }
        public byte[] Portada { get => _portada; set => _portada = value; }
        public bool Activo { get => _activo; set => _activo = value; }
    }
}

File: Examenes/EX2/22-1/BackEnd_JAVA/ConferenceSoftModel/src/pe/edu/pucp/confesoft/model/Evento.java
package pe.edu.pucp.confesoft.model;
import java.util.ArrayList;
import java.util.Date;
public class Evento {
    private int idEvento;
    private ArrayList<Ponencia> ponencias;
    private String nombre;
    private Date fecha;
    private String descripcion;
    private byte[] portada;
    private boolean activo;

    public int getIdEvento() {
        return idEvento;
    }

    public void setIdEvento(int idEvento) {
        this.idEvento = idEvento;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public Date getFecha() {
        return fecha;
    }

    public void setFecha(Date fecha) {
        this.fecha = fecha;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }

    public byte[] getPortada() {
        return portada;
    }

    public void setPortada(byte[] portada) {
        this.portada = portada;
    }

    public boolean isActivo() {
        return activo;
    }

    public void setActivo(boolean activo) {
        this.activo = activo;
    }

    public ArrayList<Ponencia> getPonencias() {
        return ponencias;
    }

    public void setPonencias(ArrayList<Ponencia> ponencias) {
        this.ponencias = ponencias;
    }
    
}